### ISSUE TITLE
add barbs linewidths

### ISSUE THREAD

**Issue by wangrenz** [lukelbd/proplot#177]
Hi, Thanks for your new release.

But I forgot to add barbs `linewidths` wrapper. 

And I add follow code at  264 line of `proplot/axes/plot.py`
```python
    'barbs': {
        'colors': 'edgecolors',
        'linewidths': 'linewidths',
        'linestyles': 'linestyles',
    },

```

Finally, it's not work.

how to solve it, Thank you.

**Comment by lukelbd** [comment]
Thanks for the report! Not sure why your attempt did not work, but this is now fixed (54c7d8c)

The dictionary you tried to edit translates generic, standardized keyword arguments for color/line width/line style into the method-specific keyword arguments (e.g. `color`, `colors`, `edgecolor`, and `edgecolors` are all equivalent). This makes things consistent across the different functions.

Here's an example:

```python
import proplot as plot

# Sample data
state = np.random.RandomState(51423)
N = 10
scale = 15
x = y = np.arange(N)
z1 = state.rand(N, N) * scale
z2 = state.rand(N, N) * scale
c = np.random.rand(N, N)

# Example figure
fig, axs = plot.subplots(ncols=2)
axs[0].barbs(z1, z2, length=5, lw=0.5, color='red')
axs[1].barbs(z1, z2, c, length=5, lw=0.5, cmap='blues', colorbar='b')
axs.format(suptitle='Barbs example')
```

![tmp](https://user-images.githubusercontent.com/19657652/84558782-739db580-acf2-11ea-8747-b14240d95305.png)

**Comment by lukelbd** [comment]
The development version can be installed with: `pip install git+https://github.com/lukelbd/proplot.git`
